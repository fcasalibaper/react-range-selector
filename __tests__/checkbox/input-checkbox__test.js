/* eslint-disable */

import Checkbox from '../../src/molecules/checkbox/index';
import renderer from 'react-test-renderer';
import React from 'react';

it('renders correctly', () => {
  const tree = renderer.create(
    <Checkbox
    inputChecked={() =>{console.log(1)}}
    isSelected={true}
    key={2}
    labelContent={'3'}
    linked={'checkboxName'}
    stateClass={'is-in-range'}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});