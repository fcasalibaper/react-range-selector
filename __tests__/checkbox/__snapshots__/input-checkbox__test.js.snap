// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="range-selector__item"
>
  <input
    checked={true}
    className="range-selector__item-checkbox"
    id="checkboxName"
    onChange={[Function]}
    type="checkbox"
    value="checkboxName"
  />
  <label
    className="range-selector__item-label is-in-range"
    htmlFor="checkboxName"
  >
    3
  </label>
</div>
`;
